test/widgets/message_list_test.dart

```diff
@@ -687,8 +688,8 @@ void main() {
       final rectsBefore = captureMessageRects(tester, messages, messageWithMarker);
       checkMarkersCount(edited: 0, moved: 0);
 
-      await store.handleEvent(eg.updateMessageMoveEvent(
-        [messageWithMarker], origTopic: 'old', newTopic: messageWithMarker.topic));
+      await store.handleEvent(eg.updateMessageEventMoveTo(
+        origTopic: 'old', newMessages: [messageWithMarker]));
```

Comment:
Hmm yeah I agree. This test needs to be refactored such that messageWithMarker is initialized in another topic, before being moved into the shared one.

Reply:
The refactor actually requires moving the message back and forth, and it has to be implemented after `message: Handle moved messages from UpdateMessageEvent.`. It wouldn't work otherwise because the move might violate the assertions (since neither topic/streamId can be updated at the time the example data helper is split).

Reply:
I see. OK, in that case then the earlier commit can just leave a comment saying yes, this data looks wrong, it'll be fixed soon.

The blocker for fixing it is basically #150, right? In that case the comment can look like `// TODO(#150): …`.

Reply:
Yeah. This is fixed in `message_list_test: Move marker message back and forth`, after move handling is implemented. I will include the TODO comment in the refactor that touches but not fixes this test.